.github/workflows/checks-vizro-ai.yml

```diff
@@ -16,7 +16,6 @@ on:
 
 concurrency:
   group: checks-ai-${{ github.head_ref }}
-  cancel-in-progress: true
```

Comment:
What would be a reason to have this 🤔 ? Are we unncessarily consuming credits if we take this out?

Reply:
If we take this out helps all builds to finish, even if some runners are busy. 
Yes, looks like in this case it will consume more credits.

Reply:
Lets go with it for now until we notice this become too much